src/Config/Database/MergedConfig.php

```diff
@@ -96,7 +103,7 @@ public function get(): array
             return $this->mergedConfig = $this->configMerger->clear($envDbConfig);
         }
 
-        if (!empty($this->connectionData->getHost())) {
+        if (!empty($this->getConnectionData()->getHost())) {
```

Comment:
`if ($value)` should be enough, `!empty()` does not bring additional value

Reply:
Yeah, you right, but I didn't want to change existence logic